src/features/messages/cards/TransactionCard.tsx

```diff
@@ -115,20 +118,29 @@ export function DestinationTransactionCard({
       </DeliveryStatus>
     );
   } else if (status === MessageStatus.Pending) {
-    content = (
-      <DeliveryStatus>
-        <div className="flex flex-col items-center">
-          <div>Delivery to destination chain still in progress.</div>
-          {isPiMsg && (
-            <div className="mt-2 text-sm max-w-xs">
-              Please ensure a relayer is running for this chain.
-            </div>
-          )}
-          <Spinner classes="my-4 scale-75" />
-          <CallDataModal debugResult={debugResult} />
-        </div>
-      </DeliveryStatus>
-    );
+    if (isDestinationEvmChain) {
+      content = (
+        <DeliveryStatus>
+          <div className="flex flex-col items-center">
+            <div>Delivery to destination chain still in progress.</div>
+            {isPiMsg && (
+              <div className="mt-2 text-sm max-w-xs">
+                Please ensure a relayer is running for this chain.
+              </div>
+            )}
```

Comment:
imo we should start to get rid of these distinctions

Reply:
Agreed, I believe someone asked me to add this a while back